rust/main/Cargo.toml

```
@@ -119,7 +96,7 @@ paste = "1.0"
 pretty_env_logger = "0.5.0"
 primitive-types = "=0.12.1"
 prometheus = "0.13"
-protobuf = "*"
+protobuf = "2.28.0"
```

keeping as is doesn't break the build, would prefer this
```suggestion
protobuf = "*"
```

---

Unpinned ✅